evacuees/src/API/EMBC.Registrants.API/EvacuationsModule/EvacuationAutoMapperProfile.cs

```diff
@@ -91,9 +96,16 @@ public EvacuationAutoMapperProfile()
                 .ForMember(d => d.HaveSpecialDiet, opts => opts.MapFrom(s => false))
                 .ForMember(d => d.SpecialDietDetails, opts => opts.MapFrom(s => new string(string.Empty)))
                 .ForMember(d => d.HasPetsFood, opts => opts.MapFrom(s => false))
-                .ForMember(d => d.FamilyMembers, opts => opts.MapFrom(s => new List<PersonDetails>()))
+                .ForMember(d => d.HouseholdMembers, opts => opts.MapFrom(s => new List<HouseholdMember>()))
```

Comment:
I don't think this member mapping is needed

Reply:
I added it as the validation test was complaining about its absence.